### PR TITLE
Added carp chair!

### PR DESCRIPTION
**Pull request by Stanislav4ix**
## About the PR
I added a carp chair for true aristocrats. :trollface:

## Why / Balance
We need more chairs! And seriously, for better RP or events.

## Technical details
It's made of carp skin, SOOOO EXPENSIVE.

## Media
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: Stanislav4ix
- add: Added carp chair!

### CODE REVIEW

**Comment by Ilya246** [comment]
what do we get next? a carp-et?

**Comment by lzk228** [comment]
> what do we get next? a carp-et?

a carp pet :trollface:

File: Resources/Prototypes/Entities/Structures/Furniture/chairs.yml [hunk 1]
```diff
@@ -349,3 +349,20 @@
...
+  id: CarpChair
+  parent: SeatBase
+  description: It looks comfy and... Carpy?..
+  components:
+  - type: Transform
+    anchored: true
+  - type: Physics
+    bodyType: Static
+  - type: Anchorable
+  - type: Rotatable
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Furniture/chairs.yml:363]: are these 2 necessary id think theyre inherited

File: Resources/Prototypes/Entities/Structures/Furniture/chairs.yml [hunk 2]
```diff
@@ -349,3 +349,20 @@
...
+  - type: Sprite
+    state: chair-carp
+  - type: Construction
+    graph: Seat
+    node: chairCarp
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Furniture/chairs.yml:368]: you have to actually make the construction graph

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Emisse** [comment]
this should probably be converted to the sheet meister system as like carp leather or something

**Comment by JoeHammad1844** [comment]
we need carp scales to drop from butchered carp (for crafting these and the carp suit)

**Comment by deltanedas** [comment]
and carp teeth to make statue now that expeds dont use faction structures :trollface:

File: Resources/Textures/Structures/Furniture/chairs.rsi/meta.json [hunk 3]
```diff
@@ -90,6 +90,10 @@
     {
       "name": "steel-bench",
       "directions": 4
+    },
+    {
+      "name": "chair-carp",
+      "directions": 4
```
Comment (ficcialfaint) [inline Resources/Textures/Structures/Furniture/chairs.rsi/meta.json:96]: copyright field

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

**Comment by TheShuEd** [comment]
Are you planning to finish your pr?

**Comment by Stanislav4ix** [comment]
> Are you planning to finish your pr?

Yes, just I'm "green", it's difficult for me

**Comment by TheShuEd** [comment]
If you have any questions, feel free to ask me here or on discord @eshhhed

**Comment by metalgearsloth** [comment]
Ping if you come back to this.

**Pull request closed without merging**